metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -152,7 +153,11 @@
                    [conn db]
                    (let [conditions [`(>= :id ~start-index)
                                      `(< :id ~(+ start-index batch-size))]
-                         _ (info "Finding batch for provider" provider-id "from id >=" start-index " and id <" (+ start-index batch-size))
+                         _ (info (format "Finding batch for provider [%s] concept type [%s] from id >= %s and id < %s"
```

Comment:
This will make it apparent in the log which concept type CMR is batch processing, i.e. is it processing collections or granules? You could update your chart to be more accurate on where the bulk indexing is at.